### PR TITLE
CORE-675: added tables to track generalized requests that require adm…

### PR DESCRIPTION
**Pull request by slr71**
…inistrative action

### CODE REVIEW

**Review by johnworth — APPROVED** [review]
:shipit:

**Comment by slr71** [comment]
Thanks for the review!

File: src/main/conversions/v2.31.0/c2_31_0_2019121201.clj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  (:use [kameleon.sql-reader :only [load-sql-file]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.31.0:20191212.01")
+
+(defn- add-requests-tables
+  "Adds database tables for requests that require administrative intervention."
+  []
+  (mapv load-sql-file
```
Comment (psarando) [inline src/main/conversions/v2.31.0/c2_31_0_2019121201.clj:11]: 👍

**Comment by slr71** [comment]
Thanks for the reviews!

**Pull request merged**